registration/registration-services/src/main/java/io/mosip/registration/dao/impl/UserOnboardDAOImpl.java

```diff
@@ -302,6 +309,27 @@ public String insertExtractedTemplates(List<BIR> templates) {
 				bioMetrics.setCrBy(SessionContext.userContext().getUserId());
 				bioMetrics.setCrDtime(Timestamp.valueOf(DateUtils.getUTCCurrentDateTime()));
 				bioMetrics.setIsActive(true);
+				
+				try (InputStream xsd = getClass().getClassLoader().getResourceAsStream("cbeff.xsd")) {
+					BiometricRecord biometricRecord = new BiometricRecord();
+					biometricRecord.setOthers(new HashMap<>());
+					biometricRecord.setSegments(Arrays.asList(template));
+					
+					CbeffContainerImpl cbeffContainer = new CbeffContainerImpl();
```

Comment:
Instead can we use this method https://github.com/mosip/packet-manager/blob/73ae96aceefadc6360cd874828955d73a0f9c205/commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/util/PacketManagerHelper.java#L67
so that we dont have to worry about compatibility issues in future.

Reply:
Modified as suggested.